### PR TITLE
Prepare for publish

### PR DESCRIPTION
**Pull request by njooma**
Removed CHANGELOG.md because we don't keep that up to date

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ dependencies:
   logger: ^1.3.0
   fixnum: ^1.1.0
   flutter_joystick: ^0.0.3
+  collection: ^1.17.1
```
Comment (clintpurser) [inline pubspec.yaml:20]: I can't tell if this is being used in your changes, its not as far as I can tell, do you plan on using it soon?

Comment (njooma) [reply pubspec.yaml:20]: It was already being used in WebRTC, but it was never depended on explicitly

Comment (clintpurser) [reply pubspec.yaml:20]: ah i see. [here](https://github.com/viamrobotics/viam-flutter-sdk/blob/bde0240b5a44acfdf49843546281e6838970b70c/lib/src/rpc/web_rtc/web_rtc_transport_stream.dart#L3) 

sweet, thanks for cleaning up.

**Review by clintpurser — COMMENTED** [review]
belated shipit

**Pull request merged**